app/models/authorization.rb

```diff
@@ -39,6 +39,10 @@ def self.omniauth_image(auth)
     end
   end
 
+  def self.external_providers
+    return self.where('provider.ne': 'tomatoes')
```

Comment:
@potomak about return, I'm writing too much python these days :)

I've used a scope as you suggested.